src/AutoMapper/Configuration/ConfigurationValidator.cs

```diff
@@ -6,34 +6,21 @@
 using AutoMapper.Internal.Mappers;
 namespace AutoMapper.Configuration
 {
-    using Validator = Action<ValidationContext>;
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class ConfigurationValidator
+    public readonly record struct ConfigurationValidator(IGlobalConfigurationExpression Expression)
```

Comment:
I've abused records for the convenient syntax, but other than some extra IL at compile time, I don't see much of a downside :) I guess I want a primary constructor.